src/openrct2/platform/platform.h

```diff
@@ -35,6 +35,10 @@
 #define MAX_PATH 260
 #endif
 
+#ifndef MAX_COMMAND_LENGTH
+#define MAX_COMMAND_LENGTH 2 * MAX_PATH
```

Comment:
Overall I like this PR, but I'd like to leave some more comments before this gets merged. Sorry it comes so late.

The `MAX_COMMAND_LENGTH` is guarded by an `#ifdef`, which means we anticipate it to be defined somewhere else. I don't think this is a valid assumption, especially in case we haven't checked it meets our length requirements.

Perhaps "scoping" it as `OPENRCT2_MAX_COMMAND_LENGTH` would be better. Unlike `MAX_PATH` (well, I do realise it's not 100% the case), it is limited to our project only. Furthermore, it seems to be limited to `linux.c` only, not even `posix.c`, so moving it there would make sense.

The macro should also be guarded with braces: `#define MAX_COMMAND_LENGTH (2 * MAX_PATH)`.